### DISCUSSION TITLE
Associating a person with an existing user

### DISCUSSION THREAD

**Discussion by ildar170975** [home-assistant/frontend#19127] [Dashboard]
Assume there are `person` entities - `jane`, `john`, ...
And there is only one user - "John Smith" associated with `person.john`.
How can we add a new user "Jane Smith" associated with `person.jane`?

Currently a `person` entity may be associated with a NEW user only:

1. Open a window to configure a `person` entity.
Toggle "Allow person to login":
![image](https://github.com/home-assistant/frontend/assets/71872483/3d315f0c-2675-4fba-8363-03427b56409d)

2. A window to create a new user is opened:
![image](https://github.com/home-assistant/frontend/assets/71872483/121c57a3-ce91-4be4-87e9-ed44f7b0e9f1)
where it is not possible to select an existing user.

There is a possible way - but in yaml only:

1. Find out a `user_id` of some user:
![image](https://github.com/home-assistant/frontend/assets/71872483/34cffb15-df1f-46a9-8f7d-e18f0021e948)

2. Add this ID to an existing `person` entity:
```
person:
  - name: Jane
    id: jane
    user_id: b2e8d127662949ef924a34239302ae42
```

**Comment by steverep** [comment]
Duplicate of #9811
